pkg/pillar/cmd/volumemgr/context.go

```diff
@@ -18,15 +18,7 @@ func (ctx *volumemgrContext) subscription(topicType interface{}, objType string)
 	}
 	switch typeName := topicType.(type) {
 	case types.ContentTreeConfig:
-		switch objType {
-		case types.AppImgObj:
-			sub = ctx.subContentTreeConfig
-		case types.BaseOsObj:
-			sub = ctx.subBaseOsContentTreeConfig
-		default:
-			log.Fatalf("subscription: Unknown ObjType %s for %T",
-				objType, typeName)
-		}
+		sub = ctx.subContentTreeConfig
```

Comment:
We are not calling the ```subscription``` function for the content tree. Can be removed from here.

Reply:
Good catch. That whole `context.go` file can be gotten rid of.